CHANGELOG.rst

```diff
@@ -29,13 +29,14 @@ Deprecated
 Added
 -----
 
+- The option ``vertical_compression`` was added to the text drawer and to the
+  ``QuantumCircuit.draw`` method. The option allows to control how much
+  room the text circuit drawing takes.
 - Introduced a visualization for the Pass Manager. (#2445)
 
 Changed
 -------
 
-- Qubits and classical bits are not represented as a tuples anymore, but as
-  instances of ``Qubit`` and ``Clbit`` respectively.
```

Comment:
Looks like another rebase mistake, I don't think we want to remove this.

Reply:
the entry seems twice in master

Reply:
Oh, I didn't realize it was duplicated there. This is fine then. Normally I'd say lets do this in a separate commit to keep with 1 logical change per commit, but the github workflow makes iterating like this more difficult.